scripts/o3de/ExportScripts/export_source_android.py

```diff
@@ -0,0 +1,383 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+import o3de.command_utils as command_utils
+import o3de.export_project as exp
+import o3de.manifest as manifest
+
+from o3de import android, android_support
+
+
+import argparse
+import logging
+import pathlib
+import platform
+import sys
+import time
+
+from typing import List
+
+def supply_android_default_global_configuration(api:int = 31,
+                                        ndk_version:str = '25*',
+                                        gradle_min_version:str = '8.1.0',
+                                        auto_store_pwd:str|None = None,
+                                        auto_key_pwd:str|None = None):
+    is_pc = exp.get_default_asset_platform() == 'pc'
+    android_sdk_home_env_var = '%ANDROID_SDK_HOME%' if is_pc else '$ANDROID_SDK_HOME'
+    android_signing_config_keystore_file_env_var = '%ANDROID_SIGNING_CONFIG_KEYSTORE_FILE%' if is_pc else '$ANDROID_SIGNING_CONFIG_KEYSTORE_FILE' 
+    android_signing_config_key_alias_env_var = '%ANDROID_SIGNING_CONFIG_KEY_ALIAS%' if is_pc else '$ANDROID_SIGNING_CONFIG_KEY_ALIAS'
```

Comment:
Something like this should be moved to [here](https://github.com/o3de/o3de/blob/8fc008743313813933cd0be190535fc4ed79a92b/scripts/o3de/o3de/android_support.py#L37-L59).

Reply:
so based on further review of your feedback, I'm inclined to agree with the idea of removing the configuration code altogether, but that also gets rid of this as well. Atm, the export script was the only place where this logic was used, so I'm not sure it's still a good idea to move this particular code into the android_support script